### PR TITLE
removed deprecated services

### PR DESCRIPTION
**Pull request by germanattanasio**
### Summary

- [x] Concept Expansion
- [x] Visual Insights
- [x] Tone Analyzer beta versions
- [x] Visual Recognition beta versions
- [x] Document Conversion beta versions

I've removed tests, examples and  updated the README. Please verify every change from the list above before merging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -565,29 +562,7 @@ tradeoff_analytics.dilemmas(params, function(err, res) {
...
-};
-
-visual_insights.summary(params, function(err, res) {
-  if (err)
-    console.log(err);
-  else
-    console.log(JSON.stringify(res, null, 2));
-});
-```
+The Watson [Visual Insights][visual_insights] Service will be withdrawn. The Watson Visual Insights Service tile will be removed from the Bluemix catalog on July 3, 2016, after which you cannot provision new instances of this service.
```
Comment (nfriedly) [inline README.md]: Do we know the sunset date for existing instances?

File: lib/index.js [hunk 2]
```diff
@@ -111,8 +111,8 @@ function createServiceAPI(serviceName) {
         if (serviceName === 'search') {
           throw new Error('The search service has been replaced by retrieve_and_rank');
         }
-        if (serviceName === 'tone_analyzer' && path.basename(version) === 'v1') {
-          throw new Error('tone_analyzer v1 has been replaced by v2-experimental');
+        if (serviceName === 'tone_analyzer' && path.basename(version) === 'v3-beta') {
+          throw new Error('tone_analyzer v3-beta has been replaced by v3');
         }
 
         // for the v1.0 breaking change of requiring experimental/beta in the module name, try to offer a helpful error message.
```
Comment (nfriedly) [inline lib/index.js]: We should probably make a more general version of this for any time the specified version isn't found. But this is good enough for today.

**Comment by nfriedly** [comment]
Concept Expansion is listed in the readme in the [ToC](https://github.com/watson-developer-cloud/node-sdk/blob/4bdc68c8056dbb003106fb45fe974bfc0ba4f955/README.md#table-of-contents) and as the example in the [Running in Bluemix](https://github.com/watson-developer-cloud/node-sdk/blob/4bdc68c8056dbb003106fb45fe974bfc0ba4f955/README.md#running-in-bluemix) section

**Comment by nfriedly** [comment]
There's still a [document conversion v1-experimental integration example](https://github.com/watson-developer-cloud/node-sdk/blob/master/examples/document_conversion_integration.v1-experimental.js) that should be removed

File: README.md (hunk 1 above)
Comment (germanattanasio) [inline README.md]: I couldn't find the date

Comment (nfriedly) [inline README.md]: Ok, then this is good enough for now.

**Pull request merged**